core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -332,8 +337,9 @@ protected void drawLaser(Tile tile, Tile target){
 
         float fract = 1f-tile.entity.power.graph.getSatisfaction();
 
-        Draw.color(Color.white, Pal.powerLight, fract*0.86f + Mathf.absin(3f, 0.1f));
-        Drawf.laser(laser, laserEnd, x1, y1, x2, y2, 0.3f);
+        Draw.color(Pal.powerLight, Pal.powerLightLowPower, fract*0.86f + Mathf.absin(3f, 0.1f));
```

Comment:
Why did you make the lasers even darker when at low power? It looks very muddy and ugly in your screenshot; I don't like it at all.

Reply:
Actually I agree...

Reply:
Reverted to original colors for now.